### DISCUSSION TITLE
PR Review Party 2 - Feedback

### DISCUSSION THREAD

**Discussion by adityaoberai** [appwrite/appwrite#2219] [General]
On 14/10/2021, we ran our second **PR Review Party** on our [Discord server](https://appwrite.io/discord) 🥳  In a 1.5-hour long session, two of our developers went live and showed you how we review your Hacktoberfest pull requests.

### We are really happy to host this session for our community and we would like to hear your opinion!

Please share your feedback and let's discuss these topics:

- Overall, did you like our session?
- What can we do to improve the session?
- Would you like to see more of these?

**Comment by m1ga** [comment]
I like the idea of those sessions but I think it could be improved a bit to make the sessions a bit more productive:

**Make the actual review part offline**. Go over some PRs beforehand and pick some that are either ready to be merged or have an issue you want to present on stream (like some common mistakes). This would also remove the `please take my PR` message in the chat of people not understanding that you have so many issues and have to be patient. 
Looking at PRs with the audience for the first time and taking "easy" issues like the copy&paste the guidelines to another repo and finding that even those have issues takes a lot of time and is a bit disappointing to see. Using just PRs you already know and can either merge directly and show which parts can be improved would help more people to make better PRs in the future!

It will be also easier for you as a reviewer so you have your time to make the actual review and takes the pressure away of 20 people watching you reading a file and finding errors :smile: 

And perhaps Discord is not the right platform for the streams. Maybe **Youtube Live or Twitch** would be better. I'm not sure about a quick and easy switch between presenters. But it would help to get more people to watch those streams. Not everyone is using Discord and I would rather see and hear just you and not random people who didn't mute their microphone :smile:
